Sources/Regex.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 class Regex {
 
     static let imagePattern = "(.+?)\\.(gif|jpg|jpeg|png|bmp)$"
+    static let openGraphImagePattern = "(.+?)\\.(gif||jpg|jpeg|png|bmp)$"
```

Comment:
Why is this separate from `imagePattern` and why is there a double `|` here?  Does it make sense to define a special pattern for all sorts of special case situations?